.github/workflows/python-package.yml

```diff
@@ -3,7 +3,7 @@
 
 name: Python package
 
-on: [pushes]
+on: [push]
```

Comment:
Very nice!
A question: How do we specify this should only runs when a pull request is merged?

Reply:
name: Python package
  on:
  pull_request:
    branches:
      - <specify the branch>

Reply:
Actually this will run when a pull_request is opened.
for running stuff when a PR is merged - see this discussion: https://github.community/t/trigger-workflow-only-on-pull-request-merge/17359